contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -50,6 +54,7 @@ def _transmit(self, envelopes):
                     'Content-Type': 'application/json; charset=utf-8',
                 },
                 timeout=self.options.timeout,
+                proxies=proxies,
```

Comment:
What happens if proxies is an empty dict? Does it do anything?

Reply:
Yes the request still passes - thanks for the suggestion - updating now.